landscape.yml

```diff
@@ -378,6 +378,11 @@ landscape:
             repo_url: https://github.com/eiffel-community/eiffel
             logo: eiffel.svg
             crunchbase: https://www.crunchbase.com/organization/eiffel-community
+          - item:
+            name: CD Events
+            description: CD Events is an event protocol providing interoperability in CD systems
+            repo_url: https://github.com/cdevents/spec
```

Comment:
I guess we only have to add ![cd events logo](https://raw.githubusercontent.com/cdfoundation/artwork/main/cdevents/horizontal/color/cdevents_horizontal-color.svg) now?

Reply:
Yes. Or rather the stacked version, as specified here: https://github.com/cdfoundation/cdf-landscape#logos